frontend/api/books.js

```diff
@@ -1,92 +1,92 @@
-import { gql, useQuery, useMutation } from "@apollo/client";
+import { gql, useQuery, useMutation } from '@apollo/client';
 
-export const useGetBooks = ( ) => {
-  const allBooksQuery = gql`
-    query getBooks {
-      getBooks {
+export const allBooksQuery = gql`
+  query getBooks {
+    getBooks {
+      id
+      title
+      coverImage
+      author {
         id
-        title
-        coverImage
-        author {
-          id
-          firstName
-          lastName
-        }
+        firstName
+        lastName
+      }
+    }
+  }
+`;
+export const aBookQuery = gql`
+  query getBook($id: ID!) {
+    getBook(id: $id) {
+      id
+      title
+      coverImage
+      author {
+        id
+        firstName
+        lastName
       }
+      description
     }
-  `;
+  }
+`;
+
+export const useGetBooks = () => {
   const { loading, error, data } = useQuery(allBooksQuery);
 
   return {
     booksLoading: loading,
     booksError: error,
-    books: data?.getBooks || [],
+    books: data?.getBooks || []
```

Comment:
The `|| []` actually causes some performance issues. Ask Jeremy about this, but it would be better to declare a static array at the top of this file `const STATIC_ARRAY = []` and use this variable as a OR return.. 

Minor nitpick!

Reply:
I'll look into that, didnt realize the performance issue. I'm guessing its because its returning an empty array (which will never not be an empty array, hence the const static declaration is more performant)...